packages/svelte/src/reactivity/index-server.js

```diff
@@ -1,5 +1,12 @@
-export const Date = globalThis.Date;
-export const Set = globalThis.Set;
-export const Map = globalThis.Map;
-export const URL = globalThis.URL;
-export const URLSearchParams = globalThis.URLSearchParams;
+export const SvelteDate = globalThis.Date;
+export const SvelteSet = globalThis.Set;
+export const SvelteMap = globalThis.Map;
+export const SvelteURL = globalThis.URL;
+export const SvelteURLSearchParams = globalThis.URLSearchParams;
+
+// Deprecated
+export const Date = SvelteDate;
+export const Set = SvelteSet;
+export const Map = SvelteMap;
+export const URL = SvelteURL;
+export const URLSearchParams = SvelteURLSearchParams;
```

Comment:
We should be consistent on the server vs the client for whether the old exports still actually work or throw an error.